packages/autocannon-ui-frontend/src/components/ResultSet.jsx

```diff
@@ -167,6 +170,84 @@ export default function ResultSet({ data }) {
                 </Table>
               </TableContainer>
             </Grid>
+            <Grid item xs={12}>
+              <Typography variant="h6" className={classes.tableHeader}>
+                <label>HTTP Statuses</label>
+                <Tooltip title="Histogram statistics about HTTP Statuses count">
+                  <HelpIcon color="primary" className={classes.helpIcon} />
+                </Tooltip>
+              </Typography>
+            </Grid>
+            <Grid item xs={12}>
+              <TableContainer component={Paper}>
+                <Table className={classes.table}>
+                  <TableHead>
+                    <TableRow>
+                      <TableCell>Type</TableCell>
+                      <TableCell>Errors</TableCell>
+                      <TableCell>Timeouts</TableCell>
+                      <TableCell>Missmatches</TableCell>
+                      <TableCell>Resets</TableCell>
+                      <TableCell>Non2xx</TableCell>
```

Comment:
since this can be deduced from the other xxx status codes, I would omit it. It's used when showing a summary in the CLI output (which does not include all the other status codes) but since we're showing them here, this is redundant